src/Bicep.LangServer/Handlers/BicepDeploymentStartCommandHandler.cs

```diff
@@ -35,7 +35,7 @@ public record BicepUpdatedDeploymentParameter(string name, string value, bool is
 
     public record BicepDeploymentStartParams(
```

Comment:
Please can you ensure you've got test coverage for the original user-reported issue?

Reply:
Tested manually for this release. Automated test will be added in a separate pr.